worker/test/src/RTC/RTCP/TestFeedbackRtpTransport.cpp

```diff
@@ -238,7 +252,14 @@ SCENARIO("RTCP Feeback RTP transport", "[parser][rtcp][feedback-rtp][transport]"
 
 		for (auto& input : inputs)
 		{
-			packet->AddPacket(input.sequenceNumber, input.timestamp, input.maxPacketSize);
+			if (&input == &inputs.front())
```

Comment:
Let's use modern C++ everywhere (here and in more places in same file):

```suggestion
			if (std::addressof(input) == std::addressof(inputs.front()))
```